client/js/socket-events/configuration.js

```diff
@@ -65,4 +65,18 @@ socket.on("configuration", function(data) {
 			// Store the "previous" value, for next time
 			$(this).data("lastvalue", nick);
 		});
+
+	$("#view-changelog").on("click", function() {
```

Comment:
Not sure this is the right place for this code, it's not really related to the configuration.

Reply:
This event is used to render windows like help, settings, etc. I just need to make changelog a view too.

Reply:
Hmmmm, I'm just not sure why this needs to happen here. The other ones exist here because they are waiting on the configuration call. This one doesn't need it.

Reply:
Actually `#view-changelog` is in the help window, hence why it needs to happen here.

Reply:
Oh yeah, you are right.